theme/static_src/src/djangonaut-space.css

```diff
@@ -118,9 +118,9 @@
 }
 
 .landing-page-footer {
-    position: fixed;
+   /* position: fixed;
```

Comment:
in this case, I will remove the position fixed to use `grow` on `main` to achieve the sticky footer